libraries/WiFi/src/WiFiClass.cpp

```diff
@@ -67,6 +67,44 @@ void WiFiClass::mode(WiFiMode_t m) {
     }
 }
 
+int WiFiClass::_beginInternal(const char* ssid, const char *passphrase, const uint8_t *bssid) {
```

Comment:
Seems OK, but could this return a `bool` since it's internal and only ever returns 0 or 1 (pass/fail)?